c10/macros/Macros.h

```diff
@@ -407,7 +407,9 @@ __host__ __device__
 // being called from device functions. Not sure if the latter works, but
 // even if not, it not being constexpr anymore should be enough to stop
 // it from being called from device code.
-// TODO This occurred in CUDA 9 (9.0 to 9.2). Test if this is fixed in CUDA 10.
+// I tried removing the following 7 lines when removing 9.2 references, but
```

Comment:
Can you file an issue with the whole message then say here that this can be deleted when that’s fixed